examples/unitTestExample/build.gradle

```diff
@@ -32,6 +32,8 @@ android {
 
 
 dependencies {
+    testCompile 'io.reactivex:rxjava:1.1.0'
```

Comment:
Ok, guessing the same issue as Jackson that JavaAssist doesn't handle well if the ClassLoader cannot find a class.